userguide/kaui/includes/tags.adoc

```diff
@@ -1 +1,127 @@
-// For the tags, it would make sense to touch on the 'system' tags versus user tags, the former (Uppercase) having a strong effect on the behavior of the system, while the latter (lowercase) having NO effect on the behavior of the system.
+=== Overview
+
+In Kaui, you can apply one or more tags to accounts and invoices, a tag being a single value. Tags in Kill Bill can be used to convey information about an object type. A tag can just provide information about an object (user tag) or it can actually affect how the object is handled in Kill Bill (control tag).
+
+This section explains how to attach tags to accounts and invoices. It also describes the steps for creating the tag definitions on which a tag is based.
+
+[NOTE]
+*Note:* You can use the API to attach tags to even more object types, such as payments and subscriptions. For more information, see the "Tags" section of the relevant object type section (for example, Payments) in the https://killbill.github.io/slate/[API Reference].
+
```

Comment:
Kaui also allows adding a user tag to a subscription. See attached screenshot.

<img width="876" alt="Subscription_Tag" src="https://user-images.githubusercontent.com/85998496/158516914-07154745-9f42-4ac8-add4-aea216cd0931.png">

Reply:
See page 76 of PDF.